lib/proxy/util.rb

```diff
@@ -71,24 +71,7 @@ def which(bin, *path)
   end
 
   def escape_for_shell(command)
```

Comment:
I considered removing this, but it's mostly used in the Puppetrun code so I'd prefer to revisit this after https://github.com/theforeman/smart-proxy/pull/637